src/views/charge/_search.php

```diff
@@ -41,6 +41,10 @@
     ]) ?>
 </div>
 
+<div class="col-md-4 col-sm-6 col-xs-12">
+    <?= $search->field('bill_id_in') ?>
+</div>
+
```

Comment:
```suggestion
```

It's counter-intuitive to search by a list of IDs
What is this filed for?

Reply:
Fist requirements of task )

Reply:
@hiqsol please explain

Reply:
it's written in the issue - to allow export charges of the given bill(s)

Reply:
It's not a user-friendly UI.

How about adding a bulk action on the bills index page? Then you can choose multiple bills with checkboxes, click on "view charges" button and get redirected to the charge index page with preset filters?